web/src/lib/utils/file-uploader.ts

```diff
@@ -83,10 +83,10 @@ async function fileUploader(
 		formData.append('assetType', assetType);
 
 		// Get Asset Created Date
-		formData.append('createdAt', createdAt);
+		formData.append('fileCreatedAt', fileCreatedAt);
 
 		// Get Asset Modified At
-		formData.append('modifiedAt', new Date(asset.lastModified).toISOString());
+		formData.append('fileModifiedAt', new Date(asset.lastModified).toISOString());
```

Comment:
Looks like this is the same value as `fileCreatedAt`. :shrug:

Reply:
Yep, I just kept the existing functionality. Potentially we can't access the createdAt here? Not really sure why it was done this way previously.